lib/cc/config/default.rb

```diff
@@ -53,10 +58,15 @@ def duplication_engine
           config: {
             "config" => {
               "languages" => %w[ruby],
-            }
+              "checks" => json.checks,
+            },
           },
         )
       end
+
+      def json
+        @json ||= JSON.new
```

Comment:
This doesn't make sense to me. `Config::X` is basically a source of configuration:

- `Default` is our standard no-config default values
- `YAML::*` come from `.codeclimate.yml`
- `JSON::*` would come from the new `.codeclimate.json`

In no case would one source reference the either, they'd all be loaded individually and possibly merged by the caller.

In the case of the default configuration, there would be no `checks` key, since the default configuration would run the engine with, well, it's own default configuration.